openmetadata-ui/src/main/resources/ui/src/components/AddDataQualityTest/components/TestCaseForm.tsx

```diff
@@ -284,7 +284,7 @@ const TestCaseForm: React.FC<TestCaseFormProps> = ({
         ]}>
         <Select
           options={testDefinitions.map((suite) => ({
-            label: suite.name,
+            label: getEntityName(suite),
```

Comment:
@TeddyCr , import is missing for the `getEntityName` method, can you take care of that?